ext/npm-utils.js

```diff
@@ -554,7 +554,7 @@ var utils = {
 			return  path.substr(0,1) === ".";
 		},
 		startsWithTildeSlash: function( path ) {
-			return path.substr(0,2) === "~/";
+			return path.substr(0,2) === "~/" || path.substr(0,2) === "@/";
```

Comment:
cache the value of `path.substr(0, 2)` in a variable rather than performing it twice.